packages/vyprvpn-deb/vyprvpn-deb.pacscript

```diff
@@ -0,0 +1,9 @@
+name="vyprvpn-deb"
+gives="vyprvpn"
+repology=("project: ${gives}")
+version="1.8"
+url="https://support.vyprvpn.com/hc/article_attachments/360054448971/${gives}-linux-cli-${version}.amd64.deb"
+replace="${gives}"
+description="VyprVPN's CLI tool"
```

Comment:
Explain what vypr is, instead of saying it's a tool.

Reply:
It's VyprVPN's CLI tool. The CLI tool of VyprVPN. What's else to explain? Lmao